gymnasium/experimental/wrappers/rendering.py

```diff
@@ -294,23 +294,21 @@ class HumanRenderingV0(gym.Wrapper):
         >>> wrapped = HumanRenderingV0(env)
         >>> obs, _ = wrapped.reset()     # This will start rendering to the screen
 
-    The wrapper can also be applied directly when the environment is instantiated, simply by passing
-    ``render_mode="human"`` to ``make``. The wrapper will only be applied if the environment does not
-    implement human-rendering natively (i.e. ``render_mode`` does not contain ``"human"``).
+        The wrapper can also be applied directly when the environment is instantiated, simply by passing
```

Comment:
I would check through creating the website, see the docs readme.md but I think that code needs to be tabs 1 more than text therefore, you might need to revert this change

Reply:
I have builded the website locally and it does seem to work. 
Do you mean it is supposed to throw new errors/warnings or that the indentation would be bad in the doc ?

Reply:
# Here is the current section in doc:

![image](https://user-images.githubusercontent.com/16002514/213870546-a096c02e-3a5c-4df0-8261-15cb7a27f2da.png)

# Here is the local section modified by this commit:

![image](https://user-images.githubusercontent.com/16002514/213870523-215688fd-1136-4567-8944-4419da2d5695.png)